src/EditorFeatures/CSharpTest/Diagnostics/Suppression/SuppressionTests.cs

```diff
@@ -1396,304 +1396,6 @@ class Class { }
             }
         }
 
-        public abstract class CSharpLocalSuppressMessageSuppressionTests : CSharpSuppressionTests
-        {
```

Comment:
No they were deleted. We no longer support generating a Local suppress message attribute.